test/print_test_stats.py

```diff
@@ -77,25 +79,28 @@ def parse_reports(folder):
             tests_by_class[class_name].append(test_case)
     return tests_by_class
 
-def build_message(test_suite):
+
+def build_message(test_case):
     return {
         "normal": {
             "build_pr": os.environ.get("CIRCLE_PR_NUMBER"),
             "build_tag": os.environ.get("CIRCLE_TAG"),
             "build_sha1": os.environ.get("CIRCLE_SHA1"),
             "build_branch": os.environ.get("CIRCLE_BRANCH"),
-            "test_name": test_suite.name,
+            "test_name": test_case.name,
```

Comment:
are we sure `test_case.name == test_suit.name`? 
you can verify this by tailing the scribe category and filter our your messages.

if they dont match, you probably need to tag all 3
```
"test_name": test_suite.name,
"test_case_name": test_case.name,
"test_case_class": test_case.class_name,
```